Microsoft.Research/ManagedContract.Setup/ManagedContracts.wxs

```diff
@@ -210,43 +208,15 @@
     <PropertyRef Id="VS90_ROOT_FOLDER" />
 
     <PropertyRef Id="VS90DEVENV"/>
-    <PropertyRef Id="VS10DEVENV"/>
-    <PropertyRef Id="VS10EXPRESSDEVENV"/>
-    <PropertyRef Id="VS10_ROOT_FOLDER"/>
-    <PropertyRef Id="VS11_ROOT_FOLDER"/>
-    <PropertyRef Id="VS12_ROOT_FOLDER_EXISTS"/>
-    <PropertyRef Id="VS14_ROOT_FOLDER_EXISTS"/>
+    <PropertyRef Id="VS2010DEVENV"/>
+    <!--<PropertyRef Id="VS10EXPRESSDEVENV"/>-->
+    <PropertyRef Id="VS2010_ROOT_FOLDER"/>
+    <PropertyRef Id="VS2012_ROOT_FOLDER"/>
+    <PropertyRef Id="VS2013_ROOT_FOLDER"/>
+    <PropertyRef Id="VS14_ROOT_FOLDER"/>
```

Comment:
:memo: In later releases of WiX 3.10, this name changes to `VS2015_ROOT_FOLDER`. It's not available in the current build from NuGet, but the value here works just as well.